marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/DeployDatabaseFieldCommand.java

```diff
@@ -141,6 +142,30 @@ protected void addExistingRangePathIndexes(Fragment newProps, Fragment existingP
         }
     }
 
+    protected void addExistingPathNamespaces(Fragment newProps, Fragment existingProps) {
+        Element newNamespaces = newProps.getInternalDoc().getRootElement().getChild("path-namespaces", MANAGE_NS);
+        if(newNamespaces == null) {
```

Comment:
We know path-namespaces exists in newProps - I don't see the point of this null check here?